src/App.js

```diff
@@ -21,10 +21,12 @@ function App() {
       <Navbar />
       <Routes>
         <Route path="/" element={<Home />} />
-        <Route path="/videoListing" element={<VideoListing />} >
+        <Route path="/videoListing" element={<VideoListing />}>
           <Route path=":category" element={<VideoListing />} />
         </Route>
-        <Route path="/videoPlayer" element={<VideoPlayer />} />
+        <Route path="/videoPlayer" element={<VideoPlayer />}>
+          <Route path=":videoId" element={<VideoPlayer />} />
```

Comment:
should have Added /videoplayer/:videoId in on route itself

Reply:
structure looks good